test/timing_load_creds.c

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2020-2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#ifndef _WIN32
+# include <sys/stat.h>
+# include <openssl/pem.h>
+# include <openssl/x509.h>
+# include <openssl/err.h>
+# include <openssl/bio.h>
+# include <../include/internal/e_os.h>
+# if defined(OPENSSL_SYS_UNIX)
+#  include <sys/resource.h>
+# endif
+
+static char *prog;
+
+static void readx509(const char *contents, int size)
+{
+    X509 *x = NULL;
+    BIO *b = BIO_new_mem_buf(contents, size);
+
+    if (b == NULL) {
+        ERR_print_errors_fp(stderr);
+        exit(1);
+    }
+    PEM_read_bio_X509(b, &x, 0, NULL);
+    if (x == NULL) {
+        ERR_print_errors_fp(stderr);
+        exit(1);
+    }
+    X509_free(x);
+    BIO_free(b);
+}
+
+static void readpkey(const char *contents, int size)
+{
+    BIO *b = BIO_new_mem_buf(contents, size);
+    EVP_PKEY *pkey;
+
+    if (b == NULL) {
+        ERR_print_errors_fp(stderr);
+        exit(1);
+    }
+    pkey = PEM_read_bio_PrivateKey(b, NULL, NULL, NULL);
+    if (pkey == NULL) {
+        ERR_print_errors_fp(stderr);
+        exit(1);
+    }
+
+    EVP_PKEY_free(pkey);
+    BIO_free(b);
+}
+
+static void print(const char *what, struct timeval *tp)
```

Comment:
Not a fan of the generic `print` name, perhaps `print_timeval`?

Reply:
Yep, changed.